fusequery/query/src/optimizers/optimizer_group_push_down.rs

```diff
@@ -32,8 +32,9 @@ impl IOptimizer for GroupByPushDownOptimizer {
         let mut rewritten_node = PlanNode::Empty(EmptyPlan {
             schema: Arc::new(DataSchema::empty())
         });
-
+        println!("{:?}", plan);
```

Comment:
Hmm, println is for debug, we shoul remove them all.

Reply:
Deleted